metadeploy/consumers.py

```diff
@@ -90,6 +96,8 @@ def is_known_model(self, model):
     def has_good_permissions(self, content):
         if self.handle_org_special_case(content):
             return True
+        if self.handle_scratch_org_special_case(content):
```

Comment:
@wlonk It looks like you've addressed the `org` special case, but I think we still want something similar for `preflightresult`, `job`, and `scratch_org`.

Reply:
I think this is now handled by the changes in `subscribable_by`.